src/serlo-editor/plugins/box/editor.tsx

```diff
@@ -35,7 +35,13 @@ export function BoxEditor(props: BoxProps) {
 
   const showToolbar = focused || isTitleFocused
 
-  if (hasNoType) return <TypeChooserBox typeState={type} />
+  if (hasNoType) {
+    return (
+      <div data-qa="plugin-box">
+        <TypeChooserBox typeState={type} />
+      </div>
```

Comment:
maybe this can be removed since you gave the wrapping `figure` in the component `data-qa="plugin-box-initial-type-chooser"` above?

Reply:
I needed both `data-qa="plugin-box"` and `data-qa="plugin-box-initial-type-chooser"` to check if the box plugin is still visible but the type chooser is not.